src/python-bindings/schedd.cpp

```diff
@@ -1553,12 +1553,11 @@ struct Schedd {
         KeyCacheEntry *k = NULL;
         int ret = 0;
 
-        // IMPORTANT: this hashtable returns 0 on success!
-        ret = (SecMan::command_map).lookup(cmd_map_ent, session_id);
-        if (ret)
-        {
+		auto command_pair = SecMan::command_map.find(cmd_map_ent);
+		if (command_pair == SecMan::command_map.end()) {
```

Comment:
Fix the space/tab use to be consistent. I suggest changing the return line to use tabs.